.github/dependabot.yml

```diff
@@ -23,3 +23,6 @@ updates:
       prefix: 'npm'
       include: 'scope'
     versioning-strategy: increase
+    ignored_updates:
```

Comment:
Are you sure that's right?
In our project we use:

```
ignore:
      - dependency-name: ''
```

Reply:
Here are the GitHub docs: https://docs.github.com/en/github/administering-a-repository/configuration-options-for-dependency-updates#ignore

Reply:
good question, I think I followed the docs correctly..
https://dependabot.com/docs/config-file/#ignored_updates

Reply:
I'm pretty sure the Dependabot config (and docs) are the same as the Dependabot config/docs for GitHub. 
As far as I know the Github dependabot as less configs.